Makefile

```diff
@@ -37,10 +37,10 @@ dockersetup: my.env .docker-build
 lint: my.env
 	${DC} run processor flake8
 
-dockertest: my.env
+dockertest: my.env .docker-build
 	./docker/run_tests_in_docker.sh ${ARGS}
 
-dockertestshell: my.env
+dockertestshell: my.env .docker-build
```

Comment:
These dependencies should have been there before. It's hard to run the tests without things being built.